sdcm/tester.py

```diff
@@ -1974,12 +1974,19 @@ def run_fullscan_thread(self, ks_cf='random', interval=1, duration=None):
             interval {number} -- interval between requests in min (default: {1})
             duration {int} -- duration of running thread in min (default: {None})
         """
+        sla_role_name, sla_role_password = None, None
+        if hasattr(self, "fullscan_role"):
```

Comment:
you can use walrus operator here with getattr

Reply:
changed